.gitignore

```diff
@@ -17,6 +17,8 @@ cq-provider-aws
 .cq/*
 # Local .terraform directories
 **/.terraform/*
+# Local .terraform.lock.hcl files
+**/terraform/**/local/.terraform.lock.hcl
```

Comment:
I had to run `terraform init` before `terraform apply -var="prefix=AB"`, and it generated these files.
In local dev context, I think it's ok to ignore them.

Reply:
agree
btw not all of the local envs accept the prefix var
maybe we should add it

Reply:
I can a separate issue for supporting the prefix var in all local envs so we can do it in another PR.
Does that work?